checks/raw/dependency_update_tool.go

```diff
@@ -23,16 +23,43 @@ import (
 	"github.com/ossf/scorecard/v4/clients"
 )
 
+const (
+	dependabotID = 49699333
+)
+
 // DependencyUpdateTool is the exported name for Depdendency-Update-Tool.
 func DependencyUpdateTool(c clients.RepoClient) (checker.DependencyUpdateToolData, error) {
 	var tools []checker.Tool
 	err := fileparser.OnAllFilesDo(c, checkDependencyFileExists, &tools)
 	if err != nil {
 		return checker.DependencyUpdateToolData{}, fmt.Errorf("%w", err)
 	}
-
 	// No error, return the tools.
+	if len(tools) != 0 {
+		return checker.DependencyUpdateToolData{Tools: tools}, nil
+	}
+
+	PRs, err := c.ListMergedPRs()
```

Comment:
so the main use case is someone who enabled deendabot via the settings, and scorecard currently cannot detect it?

Reply:
yes, it will check the PRs only after not finding a dependabot file in .github